intern/service/boss/volume.py

```diff
@@ -15,6 +15,12 @@
 from intern.resource.boss import ChannelResource, PartialChannelResourceError
 from intern.service.boss import BossService
 from intern.service.boss.v1.volume import VolumeService_1
+from enum import Enum
+
+class CacheMode(Enum):
```

Comment:
I make no claims about correctness, but I think this should look something like:
```python
class CacheMode(IntEnum):
    CACHE = 0
    NO_CACHE = 1
    RAW = 2

    @staticmethod
    def to_string(val):
        if val == CacheMode.CACHE:
            return 'cache'
        # etc
```

Reply:
Is there a benefit to this instead of my current set-up? It seems my method is cleaner? Open to suggestions.

Reply:
I would actually use `class CacheMode(str, Enum):` so that it behaves like a normal string

Reply:
I don't like how you compare against hard coded strings in other parts of the code. I think it should be `if mode == CacheMode.NO_CACHE:` as opposed to `if mode.value == 'no_cache:` for example.

Reply:
@derek-pryor The way I have defined my method, works exactly the same as `CacheMode(str, Enum)` would. So I am keeping it as is.